Docs/deep_linking.md

```diff
@@ -0,0 +1,169 @@
+
+
+### Log Curve Values View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/objects/:objectUid/curvevalues
+
+#### Additional params for searching
+
+If there is a need to search in result, following parameters need to be filled up:
+
+  - startIndex
+  - endIndex
+  - mnemonics
+
+StartIndex and endIndex could be numbers for depth based logs or datetime in ISO format  - "YYYY-MM-DDTHH:mm:ss.sssZ" for time based logs
+
+Samples:
+
+##### Time sample:
+?startIndex=2024-09-06T00:00:00Z&endIndex=2024-09-06T00:01:00Z&mnemonics=["mnem1","mnem2","mnem3","mnem4"]
+
+url encoded version which should be used:
+%3FstartIndex%3D2024-09-06T00%3A00%3A00Z%26endIndex%3D2024-09-06T00%3A01%3A00Z%26mnemonics%3D%5B%22mnem1%22%2C%22mnem2%22%2C%22mnem3%22%2C%22mnem4%22%5D
+
+In the sample above we combine mnemonics "mmem1", "mmem2", "mmem3", "mmen4" for the selected time interval
+
+##### Depth sample:
+
+?startIndex=1&endIndex=4&mnemonics=["mnem1","mnem2","mnem3","mnem4"]
+
+url encoded version which should be used:
+%3FstartIndex%3D1%26endIndex%3D4%26mnemonics%3D%5B%22mnem1%22%2C%22mnem2%22%2C%22mnem3%22%2C%22mnem4%22%5D
+
+### Multi Logs Curve Info List View
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/multilogs"
+
+
+#### Additional params for searching
+
+If there is a need to search in result, following parameter needs to be filled up:
+
+  - logs
+
+StartIndex and endIndex could be numbers for depth based logs or datetime in ISO format  - "YYYY-MM-DDTHH:mm:ss.sssZ" for time based logs
+
+Sample:
+
+?logs=["323f827b-c849-4b14-a1c3-ba207848f29c","701f5f00-bdb5-4937-8350-1975240ce432"]
+ 
+and url-encoded version, which should be used:
+
+?logs=%5B%22323f827b-c849-4b14-a1c3-ba207848f29c%22%2C%22701f5f00-bdb5-4937-8350-1975240ce432%22%5D
+
+In the sample above we combine 2 logs with uid "323f827b-c849-4b14-a1c3-ba207848f29c" and with uid "701f5f00-bdb5-4937-8350-1975240ce432".
+
+
+### Multi Logs Curve Values
+/servers/:serverUrl/wells/:wellUid/wellbores/:wellboreUid/objectgroups/:objectGroup/logtypes/:logType/multilogs/curvevalues"
+
+#### Additional params for searching
+
+If there is a need to search in result, following parameters need to be filled up:
+
+  - startIndex
+  - endIndex
+  - mnemonics
+
+StartIndex and endIndex could be numbers for depth based logs or datetime in ISO format  - "YYYY-MM-DDTHH:mm:ss.sssZ" for time based logs
+
+Samples:
+
+##### Time sample:
+?startIndex=2024-09-06T00:00:00.000Z&endIndex=2024-09-06T00:01:00.000Z&mnemonics={"323f827b-c849-4b14-a1c3-ba207848f29c":["mnem1"],"701f5f00-bdb5-4937-8350-1975240ce432":["mnem2"]}
+
+url encoded version which should be used:
+%3FstartIndex%3D2024-09-06T00%3A00%3A00.000Z%26endIndex%3D2024-09-06T00%3A01%3A00.000Z%26mnemonics%3D%7B%22323f827b-c849-4b14-a1c3-ba207848f29c%22%3A%5B%22mnem1%22%5D%2C%22701f5f00-bdb5-4937-8350-1975240ce432%22%3A%5B%22mnem2%22%5D%7D
```

Comment:
Remove url-encoding of parts of this (like mentioned previously)

Reply:
done